utils/config-schema.json

```diff
@@ -99,6 +111,63 @@
               "$ref": "#/definitions/ethaddress"
             }
           }
+        },
+        "paraswap": {
+          "type": "object",
+          "properties": {
+            "contract": {
+              "$ref": "#/definitions/ethaddress"
+            },
+            "authorisedExchanges": {
+              "type": "object",
+              "properties": {
+                "Kyber": {
```

Comment:
Could we keep this empty? Kyber was just used as a placeholder of an example Dex in `dev`.